.github/workflows/modern-ci.yml

```
@@ -0,0 +1,202 @@
+name: Modern CI (Simplified)
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)